include/imex/Dialect/GPUX/IR/GPUXOps.td

```diff
@@ -168,6 +168,15 @@ def GPUX_LaunchFuncOp
                    CArg<"::mlir::Type", "nullptr">:$asyncTokenType,
                    CArg<"::mlir::ValueRange", "{}">:$asyncDependencies)>
   ];
+
+  let extraClassDeclaration = [{
```

Comment:
Why the name is "extraClassDeclaration"? Could there possibly be a name that explains the purpose of the class?

Reply:
This is mainly to auto generate certain methods with dialect class definition, see more details in mlir doc - https://mlir.llvm.org/docs/DefiningDialects/#extra-declarations. agree that the name does not give much insight into the intended usage. Can add some detail from mlir doc.

Reply:
Generally I suggest having a "summary" and "description" in each op-def. This will auto-populate the generated docs. Without, the docs only list the ops without any description.